### ISSUE TITLE
Windows 7/8/8.1 support

### ISSUE THREAD

**Issue by Alex313031** [BigBoiCJ/SteamAutoCracker#35]
@BigBoiCJ  Hi, I make a number of forks of apps for Windows 7, including Chromium > https://thorium.rocks/win7 and some electron apps.

In light of steam removing support for Win7 on jan 1 2024, and the fact that one of the games that I have on there is ONLY available on steam, and that I have paid for it on mobile as well as steam, AND that it has graphical issues on Windows 10 and above, I am looking to this to be able to run it in the future.

Will this run on windows 7 SP1 64 bit, or will I have to install steam on my Win10 drive, crack the game, and then copy the cracked game to my Win7 drive?

If not win7, will it work on 8.0 or 8.1/server 2012?

**Comment by BigBoiCJ** [comment]
I cannot guarantee that the .exe will run on windows 7, you'll have to test. I think it should but I'm not sure how it works.

The pyinstaller module says it only officially supports Windows 8+, but 7 should work.

If it doesn't, you can still run the python script by yourself, without the bundled exe. The required libraries are located here: https://github.com/BigBoiCJ/SteamAutoCracker#requirements

For the `steam_api` emulators, my guess is that they should work fine with Win 7. Can't know for sure but I think it's all good :)

**TL;DR Windows 8+ support should be guaranteed, but Windows 7 probably works fine**
